src/fabric/fabricHelper.ts

```diff
@@ -146,12 +158,160 @@ export class FabricHelper {
         return result;
     }
 
+    public static async createWorkspace(
```

Comment:
Move `createWorkspace()` and `createFabricSqlDatabase()` above `fetchFromFabric()` so that similar methods are grouped.